.github/workflows/deploy.yml

```diff
@@ -0,0 +1,51 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - master
+env:
+  AWS_DEFAULT_REGION: us-east-1
+jobs:
+  test:
+    name: Run tests
+    env:
+      COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tox: [flake8, safety, py37, coveralls]
+    steps:
+      - uses: actions/checkout@master
```

Comment:
Same question here as the other... not sure if I don't understand or if this should be modified.